tests/test_remote.py

```diff
@@ -105,6 +105,43 @@ def test_fetch(mock_repo):
     assert mock_remote.fetch.called is True
 
 
+def test_fetch_prune(mock_repo):
+    """
+    GIVEN GitRep is initialized with a path and repo
+    WHEN remote.fetch is called
+    AND prune is True
+    THEN repo.remote is called
+    AND fetch is called with prune
+    """
+    mock_remote = Mock()
+    mock_repo.remote.return_value = mock_remote
+
+    repo = GitRepo(repo=mock_repo)
+    repo.remote.fetch(prune=True)
+    assert mock_repo.remote.called is True
+    assert mock_remote.fetch.called is True
+    assert mock_remote.fetch.call_args[1]['prune'] is True
+
```

Comment:
We probably want to check that prune_tags is not in the list of arguments as well, to make sure it doesn't get sent by mistake.